CONTRIBUTING.rst

```diff
@@ -132,9 +132,10 @@ In order to release a new version of Taskgraph, you will need permission to the
 5. Checkout the latest public revision ``git checkout main``
 6. Pull latest revision ``git pull upstream main``
 7. Verify ``git show`` outputs the desired revision
-8. Remove previously packaged releases ``rm -rf ./dist/*``
-9. Package the app ``python setup.py sdist bdist_wheel``
-10. Upload to PyPI using `twine`_ ``twine upload dist/*`` providing your
+8. Tag and push: ``git tag -s -m "taskgraph $version" $version && git push origin $version``
```

Comment:
We use `upstream` as the remote in prior steps.

Reply:
Woops, thanks for pointing that out.